virtual_ship/expedition/input_data.py

```diff
@@ -0,0 +1,173 @@
+        :param load_ctd: Whether to load the CTD fieldset.
+        :param load_drifter: Whether to load the drifter fieldset.
+        :param load_ship_underwater_st: Whether to load the ship underwater ST fieldset.
+        :returns: An instance of this class with loaded fieldsets.
+        """
+        if load_drifter:
+            drifter_fieldset = cls._load_drifter_fieldset(directory)
+        else:
+            drifter_fieldset = None
+        if load_argo_float:
+            argo_float_fieldset = cls._load_argo_float_fieldset(directory)
+        else:
+            argo_float_fieldset = None
+        if load_adcp or load_ctd or load_ship_underwater_st:
+            default_fieldset = cls._load_default_fieldset(directory)
+        if load_adcp:
+            adcp_fieldset = default_fieldset
+        else:
+            adcp_fieldset = None
+        if load_ctd:
+            ctd_fieldset = default_fieldset
+        else:
+            ctd_fieldset = None
+        if load_ship_underwater_st:
+            ship_underwater_st_fieldset = default_fieldset
+        else:
+            ship_underwater_st_fieldset = None
+
+        return InputData(
+            adcp_fieldset=adcp_fieldset,
+            argo_float_fieldset=argo_float_fieldset,
+            ctd_fieldset=ctd_fieldset,
+            drifter_fieldset=drifter_fieldset,
+            ship_underwater_st_fieldset=ship_underwater_st_fieldset,
+        )
+
+    @classmethod
+    def _load_default_fieldset(cls, directory: str | Path) -> FieldSet:
+        filenames = {
+            "U": directory.joinpath("default_uv.nc"),
+            "V": directory.joinpath("default_uv.nc"),
+            "S": directory.joinpath("default_s.nc"),
+            "T": directory.joinpath("default_t.nc"),
+        }
+        variables = {"U": "uo", "V": "vo", "S": "so", "T": "thetao"}
+        dimensions = {
+            "lon": "longitude",
+            "lat": "latitude",
+            "time": "time",
+            "depth": "depth",
+        }
+
+        # create the fieldset and set interpolation methods
+        fieldset = FieldSet.from_netcdf(
+            filenames, variables, dimensions, allow_time_extrapolation=True
+        )
+        fieldset.T.interp_method = "linear_invdist_land_tracer"
+        fieldset.S.interp_method = "linear_invdist_land_tracer"
+
+        # make depth negative
+        for g in fieldset.gridset.grids:
+            if max(g.depth) > 0:
+                g.depth = -g.depth
+
+        # add bathymetry data
+        bathymetry_file = directory.joinpath("bathymetry.nc")
+        bathymetry_variables = ("bathymetry", "deptho")
+        bathymetry_dimensions = {"lon": "longitude", "lat": "latitude"}
+        bathymetry_field = Field.from_netcdf(
+            bathymetry_file, bathymetry_variables, bathymetry_dimensions
+        )
+        # make depth negative
+        if max(bathymetry_field.data > 0):
+            bathymetry_field.data = -bathymetry_field.data
```

Comment:
Could it be that users provide bathymetry that has both positive (for land mountains) and negative (for ocean throughs) values? In that case, this test is not very useful. Perhaps better to check if `min(bathymetry_field.data >= 0)`?

Reply:
Actually, I think both checks are very likely to introduce bugs that are hard to debug for someone later. I removed the check altogether and always invert. This is valid because we only support one specific dataset at this point which needs to be inverted. When this class is made more generic, inverting can be an option.